site/content/docs/5.0/customize/sass.md

```diff
@@ -52,13 +52,13 @@ In your `custom.scss`, you'll import Bootstrap's source Sass files. You have two
 // Custom.scss
 // Option B: Include parts of Bootstrap
 
-// 1. Include functions first (so you can manipulate colors, SVGs, calc, etc)
+// 1. Include functions and variables first (so you can manipulate colors, SVGs, calc, etc)
 @import "../node_modules/bootstrap/scss/functions";
+@import "../node_modules/bootstrap/scss/variables";
 
 // 2. Include any default variable overrides here
 
 // 3. Include remainder of required Bootstrap stylesheets
-@import "../node_modules/bootstrap/scss/variables";
 @import "../node_modules/bootstrap/scss/mixins";
 
```

Comment:
This is probably where we'd need to an example of map modification.

Reply:
Between line 61 and 62, right?

Maybe a simple comment there would be enough.

```
// 4. Include any map modifications here
```

What do you think?

Because I'm inexperienced with Sass, I believed `$theme-colors` was a variable, not a "map". A comment like that would make the reader think about it...

Reply:
Probably, indeed. And we could mention it in the next part, dedicated to map customization.